kuma/users/tests/test_models.py

```diff
@@ -32,7 +32,7 @@ def test_websites(self):
             twitter='http://twitter.com/lmorchard',
             github='http://github.com/lmorchard',
             stackoverflow='http://stackoverflow.com/users/lmorchard',
-            linkedin='https://www.linkedin.com/in/testuser',
+            linkedin='https://in.linkedin.com/in/testuser',
```

Comment:
Can you make the test try multiple linkedin profile field values? I.e., `https://www.linkedin.com/in/testuser`, `https://in.linkedin.com/in/testuser`, `https://www.linkedin.com/pub/testuser`